server/routes/user_screen/create.js

```diff
@@ -0,0 +1,36 @@
+const userScreen = require("../../query/user_screen");
+const express = require("express");
+const router = express.Router();
+
+router.post("/", async (req, res) => {
+  try {
+    const { screen_canvas_id, user_billboard_id, user_screen_name, description, config } = req.body;
```

Comment:
Do we want to handle the case where these could potentially be empty strings? I know we handle the `null` cases below but I just wanted to be sure.

Reply:
i guess we could, but since this is only a prototype i don't think it's such a big deal. i think we'll be hard-pressed just to get a minimal prototype working

Reply:
This would mean we every input is required and not optional? So for example if the user didn't want to add a description. It's not a priority since we mainly want to focus on the main requires but if we wanted to make it optional we can always check if it exists and if it doesn't set it as `null` and not an empty string. Since sending a `null` to postgres is acceptable since it's set to `not null = false` already.

Reply:
` const { screen_canvas_id, user_billboard_id, user_screen_name, description, config } = req.body;`
if the user didn't include one of these columns in the request then it should be undefined.

For example, if user_billboard_id is undefined then the following code will ensure that its not included in the create query (it is effectively null):
```
    if (user_billboard_id) {
      params.user_billboard_id = user_billboard_id;
    }
```